script/bulk_name_change

```diff
@@ -231,7 +231,7 @@ end
 
 parse_options
 $stdout.sync = true
-Observation.where(name: @old_name).each do |obs|
+Observation.where(name: @old_name).find_each do |obs|
```

Comment:
@mo-nathan, @pellaea Is this script still good?  May I delete it.
I think at one point there was a user accessible bulk renamer, but it was deleted.